### ISSUE TITLE
Return logits probabilities for NeuronX TGI generated tokens

### ISSUE THREAD

**Issue by dacorvo** [huggingface/optimum-neuron#639]
### Feature request

The current NeuronX TGI server does not return logits probabilities for generated tokens.

### Motivation

This makes the `best_of` parameter useless (the first sequence is always selected).

### Your contribution

Token Selector should be modified to return the next token logit probability.
